docs/logagent/releasenotes.md

```diff
@@ -0,0 +1,83 @@
+title: Logagent Release Notes
+description: Features and improvements releases for Sematext Logagent
+
+Logagent is a dynamic project, and releases are typically in a high frequency. 
```

Comment:
Logagent is a dynamic project, and releases are typically in a high frequency. => Logagent is a dynamic project with frequent releases.

Reply:
Ping @megastef ?  Or am I looking at outdated version?